app/views/categories/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container mt-5">
+  <div class="d-flex justify-content-between bg-primary text-light align-items-center">
+    <%= link_to categories_path do %>
+      <i class="bx bxs-chevron-left display-4 text-white"></i>
+    <% end %>
+    <h5 class="card-title">Add New Category</h5>
+  </div>
```

Comment:
- [ ] As raised earlier, kindly remove the top margin around this container. Designing in accordance to template shows how well we pay attention to details. ℹ️ 

<img width="259" alt="Untitled" src="https://github.com/marvin-nyalik/BudgetApp/assets/103386183/587230b1-91e5-45fd-8697-8471cae9d40e">

**Note: This is applicable to `app/views/categories/index.html.erb` and `app/views/categories/show.html.erb` files as pointed out earlier**